R/lasso-development.R

```diff
@@ -26,6 +24,38 @@
 #' F leads to removal of rows containing NULLs.
 #' @param debug Provides the user extended output to the console, in order
 #' to monitor the calculations throughout. Use T or F.
+#' @param cores Number of cores you'd like to use. Defaults to 2.
+#' @param modelName Optional string. Can specify the model name. If used, you must load the same one in the deploy step.
+#' @section Methods: 
+#' The above describes params for initializing a new lassoDevelopment class with 
+#' \code{$new()}. Individual methods are documented below.
+#' @section $new():
```

Comment:
This method appearance is a little different than the others.  The code should match the others'.
`\code{$new()}`